src/client/eppo-client.spec.ts

```diff
@@ -575,32 +580,33 @@ describe('EppoClient E2E test', () => {
 
   describe('Eppo Client constructed with configuration request parameters', () => {
     let client: EppoClient;
-    let storage: IConfigurationStore;
```

Comment:
I was finding that this variable seemed to be impacted by other tests - there is a bit of spaghetti here that I would love to unpack so things can run in a more isolated fashion.

Reply:
Interesting. Yeah test interdependency is not good.